app/views/publishers/vacancies/edit_vacancy_sections/_edit_applying_for_the_job.html.slim

```diff
@@ -1,40 +1,26 @@
 = render "publishers/vacancies/new_attributes_tag", attributes: %i[contact_email contact_number school_visits how_to_apply application_link], section: "applying_for_the_job"
 = render "publishers/vacancies/error_tag", attributes: %i[contact_email contact_number school_visits how_to_apply application_link], section: "applying_for_the_job"
 
-h2.govuk-heading-m.app-task-list__section#applying_for_the_job_heading
-  span.app-task-list__section-number = edit_vacancy_section_number(:applying_for_the_job, current_organisation)
-  = t("jobs.applying_for_the_job")
-  = govuk_link_to(t("buttons.change"), organisation_job_build_path(@vacancy.id, :applying_for_the_job), class: "change-button", "aria-label": t("jobs.aria_labels.change_applying_for_the_job"))
+= render Shared::ReviewComponent.new(id: "applying_for_the_job", title: t("jobs.applying_for_the_job"), edit_link: organisation_job_build_path(@vacancy.id, :applying_for_the_job))
 
-dl.app-check-your-answers.app-check-your-answers--short
+.review-component__body
+  = render GovukComponent::SummaryList.new do |component|
```

Comment:
I also think we might be to do `= render GovukComponent::SummaryList.new classes: "review-component__body"`